hedera-mirror-rosetta/app/domain/types/transaction.go

```diff
@@ -7,14 +7,32 @@ import (
 // Transaction is domain level struct used to represent Transaction conceptual mapping in Hedera
 type Transaction struct {
 	ID         string
-	Operations []Operation
+	Operations []*Operation
 }
 
 // FromRosettaTransaction populates domain type Transaction from Rosetta type Transaction
-func (t *Transaction) FromRosettaTransaction(rTransaction *rTypes.Transaction) *Transaction {
-	return nil // TODO Implement
+func (t *Transaction) FromRosettaTransaction(rTransaction *rTypes.Transaction) {
+	t.ID = rTransaction.TransactionIdentifier.Hash
+
+	oArray := make([]*Operation, len(rTransaction.Operations))
```

Comment:
same comment as `tArray`. I do not think it is idiomatic to use the type of the variable and can be bad for readability